docs/advanced/document-store/benchmarking.md

```diff
@@ -0,0 +1,122 @@
+# Benchmarking
+## Results
+
+The script `scripts/benchmarking.py` benchmarks DocArray's supported Document Stores in 6 different operations:
```

Comment:
User first-person.

"To understand ..., we benchmark ... in ... way. In particular, we care about ... , which lists as follows:"

Reply:
```
````{tip}
To produce the result, one can run ...
````

```